juno/assetlist.json

```diff
@@ -206,7 +206,28 @@
         "symbol": "ASVT",
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/asvt.png"
-        }
+        },
+           {
+        "description": "DAO dedicated to building tools on the Juno Network",
+        "type_asset": "cw20",
+        "address": "juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+            "exponent": 0
+          },
+          {
+            "denom": "joe",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "name": "JoeDAO",
+        "display": "JoeDAO",
```

Comment:
"display" should be "joe", since "joe" is the denom_units that will be used to show token amounts.